### ISSUE TITLE
actionmenu: axe fixes

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#1751]
Uncomment axe-core tests.

Fix issues like: 

- Elements of role none or presentation should be flagged (presentation-role-conflict)
- Certain ARIA roles must be contained by particular parents (aria-required-parent)

**Comment by jaketrent** [comment]
Not a super-simple fix. Instead of spending time on it, will put this on hold as we anticipate deprecating ActionMenu for Menu. See https://pluralsight.slack.com/archives/C5CMEKH5Z/p1623700688059500
